lib/config/config-ops.js

```diff
@@ -21,6 +21,14 @@ var lodash = require("lodash"),
 
 debug = debug("eslint:config-ops");
 
+var RULE_SEVERITY = {
+    "error": 2,
+    "warn": 1,
+    "off": 0
+};
+
+var RULE_SEVERITY_STRINGS = ["off", "warn", "error"];
```

Comment:
Or if we're concerned about the order, the other way around:

```js
var RULE_SEVERITY_STRINGS = ["off", "warn", "error"];
var RULE_SEVERITY = RULE_SEVERITY_STRINGS.reduce(function (map, str, num) {
    map[str] = num;
    return map;
}, {});
```

Reasoning: avoiding repeating same values in two different places so it is less error-prone in case these values change.